src/types.ts

```diff
@@ -5,7 +5,7 @@ type ImageCropDataFromSpec = Parameters<Spec['cropImage']>[1];
 export interface ImageCropData
   extends Omit<ImageCropDataFromSpec, 'resizeMode' | 'format'> {
   format?: 'png' | 'jpeg' | 'webp';
-  resizeMode?: 'contain' | 'cover' | 'stretch';
+  resizeMode?: 'contain' | 'cover' | 'stretch' | 'center';
```

Comment:
Could you also add the default value in the `NativeRNCImageEditor.ts` using [`WithDefault<T, DefaultValue>`](https://reactnative.dev/docs/new-architecture-library-intro#codegen-helper-types) ?

It should look sth like following:
```ts
/**
 * (Optional) the resizing mode to use when scaling the image. If the
 * `displaySize` param is not specified, this has no effect.
 */
resizeMode?: WithDefault<'contain' | 'cover' | 'stretch' | 'center' | 'cover'>;

```

Reply:
we can't use it as RN codegen fail with the next error: `Error: Union types are unsupported in structs`

Reply:
WithDefault is special case where union types are permitted

Reply:
It's imported from codegentypes helper import (the same import path as for Double or Int32)

Reply:
implementation is still limited and it doesn't work **"in structs"**